Sources/PushNotifications.swift

```diff
@@ -40,16 +59,14 @@ import Foundation
     /// - Tag: registerDeviceToken
     @objc public func registerDeviceToken(_ deviceToken: Data, completion: @escaping () -> Void = {}) {
         guard
-            let instanceId = self.instanceId,
-            let baseURL = self.baseURL,
-            let url = URL(string: "\(baseURL)/\(instanceId)/devices/apns")
+            let instanceId = Instance.getInstanceId(),
+            let url = URL(string: "https://\(instanceId).pushnotifications.pusher.com/device_api/v1/instances/\(instanceId)/devices/apns")
         else { return }
```

Comment:
Would it be good to log something here? Otherwise it looks like a silent `return` with no indication that the request didn't go through

Reply:
In this case point of failure would be invalid `instanceId` value.
I've added a simple log message, let me know what you think.